### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by spavlusieva**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,45 +1,55 @@
...
-feature, please first open an issue and link to that issue when you
-submit your pull request.
+## Opening issues
 
-If you have any questions about a possible submission, feel free to open
-an issue too.
+For bugs or enhancement requests, please file a GitHub issue unless it's
+security related. When filing a bug remember that the better written the bug is,
+the more likely it is to be fixed. If you think you've found a security
+vulnerability, do not raise a GitHub issue and follow the instructions in our
```
Comment (mriccell) [inline CONTRIBUTING.md:10]: The Remote Console is supported why can we not provide guidance to the user who encounters a security vulnerability to open an SR with Oracle Support?

File: SECURITY.md [hunk 2]
```diff
@@ -1,18 +1,38 @@
...
 
-Non-vulnerability related security issues such as new great new ideas for security features are welcome on GitHub Issues. 
+Non-vulnerability related security issues including ideas for new or improved
+security features are welcome on GitHub Issues.
 
-## Security Updates, Alerts and Bulletins
+## Security updates, alerts and bulletins
 
-Security updates will be released on a regular cadence. Many of our projects will typically release security fixes in conjunction with the [Oracle Critical Patch Update](https://www.oracle.com/security-alerts/) program. Security updates are released on the Tuesday closest to the 17th day of January, April, July and October. A pre-release announcement will be published on the Thursday preceding each release. Additional information, including past advisories, is available on our [Security Alerts](https://www.oracle.com/security-alerts/) page.
+Security updates will be released on a regular cadence. Many of our projects
```
Comment (mriccell) [inline SECURITY.md:22]: There are two different ways to address security fixes in the WebLogic Remote Console:
1. Any Security fixes required by the WebLogic Remote Console application will be released soon after the Security vulnerability has been reported/found.
2. Fixes for the WebLogic Remote Console components that run on WebLogic Server will be addressed on a quarterly basis in conjunction with other CPU fixes.

**Review by mriccell — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 2 above)
Comment (lfeigen) [reply SECURITY.md:22]: Yes, I agree.  We do not restrict ourselves to the quarterly cadence.

**Comment by spavlusieva** [comment]
@mriccell the SECURITY.md file has been updated. Could you please review it? Thank you

**Comment by LesiaChaban** [comment]
Closed due to a large number of conflicts

**Pull request closed without merging**